File: Day12/DSA/Findnumberswitheven_no_digits.cpp
Question:- https://leetcode.com/problems/find-numbers-with-even-number-of-digits/description/


class Solution {
public:
    int findNumbers(vector<int>& nums) {
        int ans = 0;
        for (int i = 0; i < nums.size(); i++) {
            int n = nums[i];
            int c = 0;
            while (n > 0) {
                n = n / 10;
                c++;
            }
            if (c % 2 == 0) {
                ans++;
            }
        }

        return ans;
    }
};

File: Day12/DSA/FindNumberswitheven_no_digits.java
Question:- https://leetcode.com/problems/find-numbers-with-even-number-of-digits/description/

Code:- 

class Solution {
    public int findNumbers(int[] nums) {

        int ans = 0;

        for (int i = 0; i < nums.length; i++) {
            int temp = nums[i];
            int counteven = 0;

            while (temp > 0) {
                temp = temp / 10;

                counteven = counteven + 1;
            }

            if ((counteven & 1) == 0) {
                ans++;
            }

        }

        return ans;
    }
}